src/main/java/uk/gov/pay/directdebit/payers/api/CreatePayerValidator.java

```diff
@@ -1,35 +1,57 @@
 package uk.gov.pay.directdebit.payers.api;
 
-import com.google.common.collect.ImmutableMap;
 import uk.gov.pay.directdebit.common.validation.ApiValidation;
 
+import java.util.Collections;
+import java.util.HashMap;
 import java.util.Map;
 import java.util.function.Function;
 
 public class CreatePayerValidator extends ApiValidation {
-    private final static String ACCOUNT_NUMBER_KEY = "account_number";
-    private final static String SORTCODE_KEY = "sort_code";
-    private final static String NAME_KEY = "account_holder_name";
-    private final static String EMAIL_KEY = "email";
-    private final static String ADDRESS_LINE1_KEY = "address_line1";
-    private final static String ADDRESS_CITY_KEY = "city";
-    private final static String ADDRESS_COUNTRY_KEY = "country_code";
-    private final static String ADDRESS_POSTCODE_KEY = "postcode";
-
-    private static boolean isNumeric(String string) {
-        return string.matches("[0-9]+");
+
+    public final static String NAME_KEY = "account_holder_name";
+    public final static String SORTCODE_KEY = "sort_code";
+    public final static String ACCOUNT_NUMBER_KEY = "account_number";
+    public final static String ADDRESS_COUNTRY_KEY = "country_code";
+    public final static String ADDRESS_LINE1_KEY = "address_line1";
+    public final static String ADDRESS_CITY_KEY = "city";
+    public final static String ADDRESS_POSTCODE_KEY = "postcode";
+    public final static String EMAIL_KEY = "email";
+
+    private final static Map<String, Function<String, Boolean>> validators;
+    static {
```

Comment:
I don't understand why we are statically initializing everything, creating a hashmap, but then switching to unmodifiableMap. Could you explain, please? Thanks :)

Reply:
@simad I just wanted to use the native approach to create immutable map and not depend on 3rd party library, as described here: https://stackoverflow.com/questions/22636575/unmodifiablemap-java-collections-vs-immutablemap-google/22636750#comment65774752_22636750
as discussed I will revert